c3/index.d.ts

```diff
@@ -1,10 +1,17 @@
-// Type definitions for C3js
+// Type definitions for C3js v0.4.11
```

Comment:
I'll remove it. As an FYI, I tended to keep the patch version in for the d3-xxxxx definitions for ttwo reasons:
(1) It's easier to go back to the underlying library and check for changes in commits after the last validated patch w.r.t. definition  accuracy, and
(2) which goes hand in hand with (1), while a patch release in JS may not break the JS API, there are times when it does break a strongly typed TS API definition.